kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/bootstrap/FilterChainFactory.java

```diff
@@ -73,6 +73,16 @@ public ScheduledExecutorService eventLoop() {
                     return null;
                 }
 
+                @Override
+                public ScheduledExecutorService filterDispatchThreadService() {
+                    return null;
+                }
+
+                @Override
+                public boolean inFilterDispatchThread() {
+                    return false;
```

Comment:
maybe we should look at splitting FilterChainFactory into an `InitialisationContext` and `FilterConstructContext`.